src/crm/getTestStation.ts

```diff
@@ -0,0 +1,34 @@
+import pRetry from 'p-retry';
+import dateFormat from 'dateformat';
+import logger from '../observability/logger';
+import { getTestStationEntities } from './dynamicsWebApi';
+import { DynamoTestStation } from '../Interfaces/DynamoTestStation';
+
+export const getTestStations = async (date: Date): Promise<DynamoTestStation[]> => {
+  const ceUrl = process.env.CE_RESOURCE;
```

Comment:
Safety check should be added if this env var has not been set

Reply:
Ideally this should be in a centralised config for this package, but this is optional